quickstart/speech-translation/java-jre/pom.xml

```diff
@@ -0,0 +1,33 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.microsoft.cognitiveservices.speech.samples</groupId>
+  <artifactId>quickstart</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <build>
+    <sourceDirectory>src</sourceDirectory>
+    <plugins>
+      <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.8.0</version>
+        <configuration>
+          <source>1.8</source>
+          <target>1.8</target>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+  <dependencies>
+	<dependency>
+	  <groupId>com.microsoft.cognitiveservices.speech</groupId>
+	  <artifactId>client-sdk</artifactId>
+	  <version>1.2.0</version>
+    </dependency>
+  </dependencies>
+  <repositories>
+	<repository>
+      <id>maven-cognitiveservices-speech</id>
+      <name>Microsoft Cognitive Services Speech Maven Repository</name>
+	  <url>https://csspeechstorage.blob.core.windows.net/maven/</url>
+	</repository>
```

Comment:
mixes tabs and space. please use spaces

Reply:
(alos above)

---
In reply to: [249396684](https://github.com/Azure-Samples/cognitive-services-speech-sdk/pull/142#discussion_r249396684) [](ancestors = 249396684)